src/FsAutoComplete.Core/FCSPatches.fs

```diff
@@ -0,0 +1,237 @@
+/// this file contains patches to the F# Compiler Service that have not yet made it into
```

Comment:
like the comment says, this is all copy/paste (with slight modifications to use idioms we have in FSAC) of code that exists in FCS in preview releases.

Reply:
Any reason why we can't use nightly FCS nuget build rather than having this here?

Reply:
Nightly FCS has some very breaking changes to sync/async APIs and would require revving fantomas/fsharplint/fake to account for binary incompatibility